warehouse/query-core/src/test/java/datawave/query/jexl/visitors/QueryFieldsVisitorTest.java

```diff
@@ -34,10 +34,6 @@ public void testEQ() throws ParseException {
         // identifiers
         query = "$12 == 'bar'";
         test(query, Collections.singleton("12"));
-        
-        // grouping context
-        query = "FOO.12 == 'bar'";
-        test(query, Collections.singleton("FOO"));
```

Comment:
Noting our previous conversation, grouping notation on query fields is explicitly NOT supported by datawave in Jexl3. For a query with grouping notation fields we may want to either fail the query or rewrite the fields.

Reply:
Just clarifying that it's not supported for jexl2 or jexl3.  jexl2 would fail silently and munge the field name.  With the jexl3 update, this actually WILL fail if you attempt to use it because our code won't be able to find the identifier.  That's why it had to be removed from the test.